test/e2e/ingress/002_ensure_v1beta1_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright Project Contour Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build e2e
+
+package ingress
+
+import (
+	"context"
+	"testing"
+
+	"github.com/projectcontour/contour/test/e2e"
+	"github.com/stretchr/testify/require"
+	"k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// Explicitly ensure v1beta1 resources continue to work.
+func testEnsureV1Beta1(t *testing.T, fx *e2e.Framework) {
+	namespace := "002-ensure-v1beta1"
```

Comment:
like you've said previously we should move this into the framework somewhere, but for now would be nice to prefix as "002-ingress-..."